wg-api/spec-documents/wc-print-sync.md

```diff
@@ -0,0 +1,46 @@
+# Print Functionality Synchronization
+
+## Summary
+
+`webContents.print()` and `webContents.printToPDF()` should pass identical parameter types in the same shape. Parameters which differ between the two will be changed to remove differences and prevent unwanted bugs. `webContents.print()` will be Promisified.
+
+## Platforms
+
+All.
+
+## Impetus
+
+`webContents.print()` and `webContents.printToPDF()` pass nearly identical sets of parameters, but given that they were implemented at different times, there is a small delta in the shape in which those parameters are passed in. Our argument sanitization for `webContents.print()` lived natively, and for `webContents.printToPDF()` lives in JavaScript. I plan to refactor these two functions to move all sanitization into JavaScript for the following primary reasons:
```

Comment:
> two functions to move all sanitization into JavaScript for the following primary reasons

Those functions should never be called in high frequency, making this move super sensible 👍